### PR TITLE
Allow artifact renaming

### PR DESCRIPTION
**Pull request by josenavas**
Built on top of #1714 so review/merge that one first.

Updates the interface to allow changing the artifact name.
Changes the colors of the study menu buttons.

### CODE REVIEW

**Comment by josenavas** [comment]
For reference, this is the new look of the menu:
screen shot 2016-03-26 at 5 33 03 pm

File: qiita_pet/templates/study_ajax/artifact_summary.html [hunk 1]
```diff
@@ -45,12 +45,52 @@
...
+      url: '/artifact/',
+      type: 'PATCH',
+      data: {'op': 'replace', 'path': '/' + artifact_id + '/name/', 'value': new_name, 'artifact_id': artifact_id},
+      success: function(data) {
+        $("#update-artifact-name").modal('hide');
+        if(data.status == 'error') {
+          bootstrapAlert(data.message, "danger");
+        }
+        else {
+          $("#summary-title").text('Summary: ' + new_name + ' (ID: ' + artifact_id + ')');
```
Comment (antgonza) [inline qiita_pet/templates/study_ajax/artifact_summary.html]: Do we really need Summary: name, (ID: bla!)? What about name (ID: bla)?

File: qiita_pet/templates/study_ajax/artifact_summary.html [hunk 2]
```diff
@@ -45,12 +45,52 @@
...
+          load_template_graph();
+        }
+      }
+    });
+      console.log('clicked ' + artifact_id + ' ' + new_name);
```
Comment (antgonza) [inline qiita_pet/templates/study_ajax/artifact_summary.html]: rm?

**Comment by antgonza** [comment]
a few comments

File: qiita_pet/templates/study_ajax/artifact_summary.html (hunk 1 above)
Comment (josenavas) [inline qiita_pet/templates/study_ajax/artifact_summary.html]: Done

File: qiita_pet/templates/study_ajax/artifact_summary.html (hunk 2 above)
Comment (josenavas) [inline qiita_pet/templates/study_ajax/artifact_summary.html]: Opps done

**Comment by antgonza** [comment]
Changes will not affect tests, merging.

**Pull request merged**